### PR TITLE
Fix Klocwork issues in Java

### PR DESCRIPTION
**Pull request by dkurt**

### CODE REVIEW

File: modules/java_api/samples/ArgumentParser.java [hunk 1]
```diff
@@ -25,24 +25,19 @@ private void printHelp() {
...
+                    input.put(arg, args[++i]);
                 } else {
-                    if (description.containsKey(arg)) {
-                        input.put(arg, args[++i]);
-                    } else {
-                        System.out.println("Non-existent key: '" + arg + "'");
-                        System.exit(0);
-                    }
+                    System.out.println("Non-existent key: '" + arg + "'");
+                    System.exit(0);
```
Comment (likholat) [inline modules/java_api/samples/ArgumentParser.java:38]: But what if we run:
```
java -cp "benchmark_app.jar" Main -m model.xml -d
```

File: modules/java_api/samples/benchmark_app/Main.java [hunk 2]
```diff
@@ -91,7 +94,7 @@ static Blob blobRandomByte(TensorDesc tDesc) {
         }
 
         byte[] buff = new byte[size];
-        Random rand = new Random();
+        SecureRandom rand = new SecureRandom();
```
Comment (likholat) [inline modules/java_api/samples/benchmark_app/Main.java:97]: Why do we need `SecureRandom` here?

Comment (dkurt) [reply modules/java_api/samples/benchmark_app/Main.java:97]: Static code analyzer stays that Random is not secure :)

File: modules/java_api/samples/ArgumentParser.java (hunk 1 above)
Comment (dkurt) [reply modules/java_api/samples/ArgumentParser.java:38]: Good point. Fixed

**Review by alvoron — APPROVED** [review]

**Pull request merged**